source/annex/macro.rst

```diff
@@ -200,7 +38,7 @@ setting the labor force index in the base year to 1 (numeraire) and solving for
 .. math:: a_r = Y_{y=0, r}^{\rho_r} - \sum_s b_{r,s} \cdot \frac{{{PHYSENE}_{r, s, y=0}}^{\rho_r}} {{K_{y=0, r}}^{\rho_r \cdot \alpha_r}}
 
 Macro-economic parameters
```

Comment:
I think the text here is important. We should probably move the listing of numerical values to a table, though (cleaner to read).